mm2src/coins/qrc20/rpc_clients.rs

```diff
@@ -0,0 +1,336 @@
+use super::*;
+use std::collections::HashMap;
+
+pub mod for_tests {
+    use super::*;
+
+    #[derive(Debug, Deserialize)]
+    pub struct ContractCreateResult {
+        /// The transaction id.
+        pub txid: H256Json,
+        /// QTUM address of the sender.
+        pub sender: String,
+        /// ripemd-160 hash of the sender.
+        pub hash160: H160Json,
+        /// Expected contract address.
+        pub address: H160Json,
+    }
+
+    #[derive(Debug, Deserialize)]
+    pub struct SendToContractResult {
+        /// The transaction id.
+        pub txid: H256Json,
+        /// QTUM address of the sender.
+        pub sender: String,
+        /// ripemd-160 hash of the sender.
+        pub hash160: H160Json,
+    }
+
+    /// QRC20 Native RPC operations that may change the wallet state.
+    pub trait Qrc20NativeWalletOps {
```

Comment:
Do we need the `Qrc20NativeWalletOps` trait? I guess there is only 1 implementation of the native RPC, so the trait looks a bit excessive to me.

Reply:
`Qrc20NativeWalletOps` is set of functions that is used in docker_tests only and should not be used in MM2. Do you offer to implement these operations as static functions, not as a trait?

Reply:
I think we can rename it for example to `Qrc20NativeTestOps`

Reply:
This specific part can stay as is I think. The only concern is the trait has a single implementor and won't ever have more of them, I think. However, this approach allows us to easily add more functionality to NativeClient.